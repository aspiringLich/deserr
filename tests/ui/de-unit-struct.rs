use deserr::DeserializeFromValue;

#[derive(DeserializeFromValue)]
#[deserr(error = deserr::Error)]
struct UnitStruct;

fn main() {}
