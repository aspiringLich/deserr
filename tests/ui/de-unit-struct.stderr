error: Unit structs aren't supported by the Deserr derive macro
 --> tests/ui/de-unit-struct.rs:3:10
  |
3 | #[derive(Deserr)]
  |          ^^^^^^
  |
  = note: this error originates in the derive macro `Deserr` (in Nightly builds, run with -Z macro-backtrace for more info)
